ocpp-common/src/main/java/eu/chargetime/ocpp/utilities/MoreObjects.java

```diff
@@ -0,0 +1,656 @@
+package eu.chargetime.ocpp.utilities;
+
+/*
+ * Copyright (C) 2014 The Guava Authors
+ * Modified by Evgeny Pakhomov <eugene.pakhomov@ubitricity.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
```

Comment:
You set the license to a different one that the project. How does that work?

Reply:
I will consult our lawyer to be on a safe side but from what I know Apache License 2 is permissive and compliant with MIT as far as its own conditions fullfilled.

Reply:
I've checked with lawyer - Apache License 2 is compatible with MIT. Additional restrictions are mostly that when you use it you have to mention changes specifically and you obliged to publish the code. So I've added detailed changes description.